x-pack/plugins/synthetics/server/routes/settings/params/add_param.ts

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { ALL_SPACES_ID } from '@kbn/security-plugin/common/constants';
+import { DEFAULT_SPACE_ID } from '@kbn/spaces-plugin/common';
+import { SavedObject, SavedObjectsBulkCreateObject } from '@kbn/core-saved-objects-api-server';
+import { SyntheticsRestApiRouteFactory } from '../../types';
+import {
+  SyntheticsParamRequest,
+  SyntheticsParams,
+  SyntheticsParamSOAttributes,
+} from '../../../../common/runtime_types';
+import { syntheticsParamType } from '../../../../common/types/saved_objects';
+import { SYNTHETICS_API_URLS } from '../../../../common/constants';
+
+const ParamsObjectSchema = schema.object({
+  key: schema.string(),
+  value: schema.string(),
+  description: schema.maybe(schema.string()),
+  tags: schema.maybe(schema.arrayOf(schema.string())),
+  share_across_spaces: schema.maybe(schema.boolean()),
+});
+
+export const addSyntheticsParamsRoute: SyntheticsRestApiRouteFactory<
+  SyntheticsParams | SyntheticsParams[]
+> = () => ({
+  method: 'POST',
+  path: SYNTHETICS_API_URLS.PARAMS,
```

Comment:
This still uses the `/internal/synthetics/params` path, which doesn't correspond to the docs, is this on purpose?